app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -3,12 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // Tells the server to emit the `flow.${viewName}.begin` event.
-// THIS MUST BE MIXED IN *AFTER* views/mixins/flow-events-mixin.
 
 define(function (require, exports, module) {
   'use strict';
 
+  const FlowEventsMixin = require('views/mixins/flow-events-mixin');
+
   module.exports = {
+    requireMixins: [
```

Comment:
Declarations with only one mixin should have a consistent style. Some use one line, others use 3.